packages/dialtone-vue2/recipes/conversation_view/editor/editor.vue

```diff
@@ -41,6 +41,7 @@
                   class="d-fw-bold"
                 />
               </template>
+              {{ button.label || '' }}
```

Comment:
hmm I think the empty string is causing the label slot to render when we don't want it to. maybe just `{{ button?.label }}` that way it would pass undefined instead of empty string when it doesn't exist.